api/spec/requests/spree/api/v2/storefront/account/credit_cards_spec.rb

```diff
@@ -40,6 +40,16 @@
         expect(json_response['data'][0]).to have_type('credit_card')
         expect(json_response['data'].size).to eq(credit_cards.count)
       end
+
+      context "user has a credit card that is not available on the front end" do
```

Comment:
@tomurb we also need test for payment methods available on front end but assigned to different store(s)